.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: ruby
 
 rvm:
-  - 2.1
+  - 2.3.1
```

Comment:
@brodock Thanks for the head up! Since all of our other gems’ `.travis.yml` use `2.1` and `2.2`, I think we should use:

- `2.1`
- `2.2`
- `2.3.1`